inventory/group_vars/OSEv3/general.yml

```diff
@@ -0,0 +1,11 @@
+---
+openshift_pkg_version: "-0.0.1"
```

Comment:
We just merged a PR into Origin that means we don't always build `origin-0.0.1` RPMs -- you'll want to test this out and see if you just want to get rid of this line, or figure out how to get the correct version in here.